.evergreen/start-docker-envs.sh

```diff
@@ -1,70 +1,77 @@
 #! /usr/bin/env bash
 
+#########################################################
+###                 !!! IMPORTANT !!!                 ###
+###  THIS SCRIPT IS EXECUTED WITH THE SOURCE COMMAND  ###
```

Comment:
I suppose we can't execute it rather than source it because that would break the trap that stops all these things?